ckanext/dalrrd_emc_dcpr/assets/js/removeDcprRequestDatasets.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+ckan.module('remoceDcprRequestDatasets', function(jQuery, _){
```

Comment:
please fix the typo on the name of the module

```suggestion
ckan.module('removeDcprRequestDatasets', function(jQuery, _){
```